src/Sarif/Core/PropertyBagHolder.cs

```diff
@@ -19,6 +19,12 @@ namespace Microsoft.CodeAnalysis.Sarif
     /// </summary>
     public class PropertyBagHolder : IPropertyBagHolder
```

Comment:
>PropertyBagHolder [](http://example.com/codeflow?start=17&length=17)

Looking forward, what would be the appropriate analog for this in Jschema? Do we have a comparable base class in that repo to extend this same type of fix to?

Reply:
Great question. We must 'go see'.